vms/platformvm/txs/disable_subnet_validator_tx.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package txs
+
+import (
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/vms/components/verify"
+)
+
+var _ UnsignedTx = (*DisableSubnetValidatorTx)(nil)
+
+type DisableSubnetValidatorTx struct {
+	// Metadata, inputs and outputs
+	BaseTx `serialize:"true"`
+	// ID corresponding to the validator
+	ValidationID ids.ID `serialize:"true" json:"validationID"`
+	// Authorizes this validator to be disabled
+	DisableAuth verify.Verifiable `serialize:"true" json:"disableAuthorization"`
+}
+
+func (tx *DisableSubnetValidatorTx) SyntacticVerify(ctx *snow.Context) error {
+	switch {
+	case tx == nil:
+		return ErrNilTx
+	case tx.SyntacticallyVerified:
+		// already passed syntactic verification
+		return nil
+	}
+
+	if err := tx.BaseTx.SyntacticVerify(ctx); err != nil {
```

Comment:
`BaseTx.SyntacticVerify` doesn't turn on the `SyntacticallyVerified` of itself. 
Since we have syntactically verified the baseTx, shall we activate it here?